### ISSUE TITLE
Missing LICENSE file

### ISSUE THREAD

**Issue by nekofar** [mimalef70/fontara#54]
Can you tell under which license is your project or add a LICENSE file??

**Comment by harubi** [comment]
Yeah, this should be looked into. There's a lot of uncertainty regarding contribution to this project.
